rust/processor/src/models/coin_models/coin_supply.rs

```diff
@@ -5,14 +5,21 @@
 #![allow(clippy::extra_unused_lifetimes)]
 #![allow(clippy::unused_unit)]
 
-use super::coin_infos::CoinInfoQuery;
-use crate::{models::default_models::move_tables::TableItem, schema::coin_supply};
+use crate::{
+    models::default_models::move_tables::TableItem,
+    processors::coin_processor::APTOS_COIN_TYPE_STR, schema::coin_supply, utils::util::hash_str,
+};
 use anyhow::Context;
 use aptos_indexer_protos::transaction::v1::WriteTableItem;
 use bigdecimal::BigDecimal;
 use field_count::FieldCount;
 use serde::{Deserialize, Serialize};
 
+const APTOS_COIN_SUPPLY_TABLE_HANDLE: &str =
+    "0x1b854694ae746cdbd8d44186ca4929b2b337df21d1c74633be19b2710552fdca";
```

Comment:
Aren't these table handles going to be different based on the environment (mainnet vs devnet)?